templates/workloads/common/cf/autoscaling.yml

```diff
@@ -54,6 +54,48 @@ AutoScalingPolicyECSServiceAverageMemoryUtilization:
       ScaleOutCooldown: 60
       TargetValue: {{.Autoscaling.Memory}}
 {{- end}}
+{{if .Autoscaling.Requests}}
+AutoScalingPolicyALBSumRequestCountPerTarget:
```

Comment:
Like mentioned above, I think these scaling policies should only be available for LBWebServices.
For backend services, we don't have a target group and the lb is not guaranteed to be created anymore, so the stack will go 💥